src/views/partials/settings.js

```diff
@@ -152,12 +158,23 @@ export const settings = data => {
 
         <hr>
 
+        <!-- Breach alert preferences -->
+        <section>
+          <h3 class='settings-section-title'>
+            ${getMessage('settings-alert-preferences-title')}
+          </h3>
+          ${alertOptions({ csrfToken, allEmailsToPrimary })}
+        </section>
+
+        <hr>
+        ${AppConstants.CANCEL_SUBSCRIPTION_FLOW ? cancelPremiumSubscription() : '<hr>'}
```

Comment:
I might be missing something here, but if `AppConstants.CANCEL_SUBSCRIPTION_FLOW` is falsy, does that mean this would stack _TWO_ `<hr>` tags here?